packages/mongodb-rag-core/src/VectorStore.ts

```diff
@@ -15,6 +15,17 @@ export type VectorStore<T> = {
 
 export type WithScore<T> = T & { score: number };
 
+/**
+  Filters for querying the vector store.
+ */
+export type QueryFilters = {
+  sourceName?: string;
+  version?: {
+    current?: boolean;
+    label?: string;
+  };
+};
+
```

Comment:
this type should be on the `EmbeddedContentStore`, not the `VectorStore`. pls move